apps/etterlatte-vedtaksvurdering/src/main/kotlin/vedtaksvurdering/database/RepositoryProxy.kt

```diff
@@ -0,0 +1,69 @@
+package no.nav.etterlatte.vedtaksvurdering.database
+
+import kotliquery.Row
+import kotliquery.Session
+import kotliquery.queryOf
+import kotliquery.sessionOf
+import kotliquery.using
+import org.slf4j.LoggerFactory
+import java.io.Serializable
+import javax.sql.DataSource
+
+class RepositoryProxy(private val datasource: DataSource) {
```

Comment:
Hvorfor ikke dra dette rett inn i repository klassen?

Reply:
Eller hva med å gjøre motsatt - trekke det ut til common og la alle repositoriene bruke samme standard? Tipper dette er første steg på veien dit

Reply:
Ooh, liker tanken på å trekkje det ut til common. Alt som ligg i denne klassa er jo veldig teknisk-generisk kode.

@sebassonav, sjå forklaringa i PR-teksten: 
Har skilt mellom det som er funksjonelt komplekst (spørringane) og det som er teknisk komplekst (kotliquery-oppbygginga) i to forskjellige klassar. Det gjorde iallfall min mentale modell og mi oversikt mykje enklare

Reply:
Ja, er _veldig_ open for betre namn på denne klassa.